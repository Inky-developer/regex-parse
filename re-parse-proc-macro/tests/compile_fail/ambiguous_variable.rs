use re_parse_proc_macro::re_parse;

fn main() {
    re_parse!("Foo{variable}B*{other_variable}C", "Foo111B222C")
}