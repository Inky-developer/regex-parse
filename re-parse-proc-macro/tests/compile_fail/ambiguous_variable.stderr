error: proc macro panicked
 --> tests/compile_fail/ambiguous_variable.rs:4:5
  |
4 |     re_parse!("Foo{variable}B*{other_variable}C", "Foo111B222C")
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: (TODO THROW ERROR) ambiguous variables: variable collides with other_variable
